spec/models/accounting/quickbooks/updater_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe Accounting::Quickbooks::Updater, type: :model do
+  let(:connection) { instance_double(Accounting::Quickbooks::Connection, last_updated_at: last_updated_at) }
+  subject { described_class.new(connection) }
+
+  let(:generic_service) { instance_double(Quickbooks::Service::ChangeDataCapture, since: double(all_types: [])) }
+  before do
+    allow(subject).to receive(:service).and_return(generic_service)
+    allow(connection).to receive(:last_updated_at=)
+  end
+
+  describe '#update' do
+    context 'with updates_since is nil' do
```

Comment:
Although I'm not seeing where the `updates_since` param is actually getting used. Can we ditch it? It's adding a lot of complexity to these tests.